src/BrutForceForCaesarCipher.java

```diff
@@ -0,0 +1,148 @@
+    private static final String MANUAL_BRUT_FORCE = "manual";
+    private static final String BRUT_FORCE_WITH_ANALYZER = "analyzer";
+
+    public static void startBrutForce(Alphabet setAlphabet){
+        alphabet = setAlphabet;
+        int usingAlphabetSize = setAlphabet.getAlphabetCodes().size();
+        System.out.println("BrutForce starting...");
+        //Get file for decode
+        System.out.println("Enter path to file which you want to decode with BrutForce");
+        Path path = WorkWithFiles.getFilePath();
+
+        //choose brut method
+        String brutForceMethod = choiseOfBrutForceMethod();
+
+        //Read data from file
+        StringBuilder decodedDataFromFile = new StringBuilder();
+        if (brutForceMethod.equals(MANUAL_BRUT_FORCE)){
+             decodedDataFromFile = startManualBrutForce(path, usingAlphabetSize);
+        } else if (brutForceMethod.equals(BRUT_FORCE_WITH_ANALYZER)){
+            decodedDataFromFile = startBrutForceWithAnalyzer(path, usingAlphabetSize);
+        } else {
+            //TODO make this exception
+            //throw invalidBrutMethodException;
+        }
+        
+        //Create and get path to file
+        Path pathToEncodingFile = WorkWithFiles.createFile(path, "Decoded_by_BrutForce");
+
+        //Write data to file
+        try {
+            Files.writeString(pathToEncodingFile, decodedDataFromFile);
+            System.out.printf("Decoded file created: %s %n", pathToEncodingFile.getFileName().toString());
+        } catch (IOException e) {
+            System.out.println("Check that the coding file in encoding UTF-8");
+            throw new RuntimeException(e);
+        }
+    }
+
+    //Manual hack
+    private static StringBuilder startManualBrutForce(Path path, int usingAlphabetSize){
+        try {
+            //Get data from file
+            List<String> dataFromFile = Files.readAllLines(path);
+            int counter = 0;
+            //Get line from data for validation
+            int lineNumberToRead = dataFromFile.size() > 1 ? 1 : 0;
+            while (counter <= usingAlphabetSize){
+                //Decode data
+                StringBuilder decodedLineFromFile = CaesarCipher.decoder(counter, dataFromFile.get(lineNumberToRead), alphabet);
+                System.out.printf("Iteration %d of %d%n".concat("Is this text correct?%n"), counter, usingAlphabetSize);
+                System.out.println("---> " + decodedLineFromFile);
+                //Manual validate result
+                String chosenOption = evaluateBrutForceResult();
+                if(chosenOption.equalsIgnoreCase("1")){
+                    break;
+                }
+                counter++;
+            }
+            //Decode all data
+            return CaesarCipher.decoder(counter, String.join("\n", dataFromFile), alphabet);
+        } catch (IOException e) {
+            System.out.println("Check that the coding file in encoding UTF-8");
+            e.printStackTrace();
```

Comment:
actually it's not a good practice to print a stack trace

Reply:
Ok, i'll delete it